Correct search term for Jetbrains Marketplace

jason-pollock opened the pull request:
Simple tweak that provides the correct search term for finding this theme in the [Jetbrains Marketplace](https://plugins.jetbrains.com/) or via Jetbrains software via `Settings -> Plugins -> Marketplace`. 

Screenshots demonstrating difference between search terms:

![2024-06-12_15-38](https://github.com/junkfactory/tokyonight-jetbrains/assets/85717231/b0c0aed4-48c6-40d3-8ec7-5108c8668240)

![2024-06-12_15-37](https://github.com/junkfactory/tokyonight-jetbrains/assets/85717231/5b23993b-5699-431c-8d94-1e86a7d90812)

jason-pollock committed:
Update README correcting Marketplace search term

wojciech-zurek commented:
I think it would be a better idea to use the name Tokyo Night Dark Theme, as users search by the name Tokyo Night

jason-pollock committed:
Update search keywords

Instruct people to search the Marketplace for "Tokyo Night Dark Theme", as "TokyoDark Theme" doesn't return any results.

jason-pollock commented:
> I think it would be a better idea to use the name Tokyo Night Dark Theme, as users search by the name Tokyo Night

Excellent. That phrase also returns your theme as part of the results. Update made in latest changes.  :100:

junkfactory commented:
Looks good to me. I'm terrible at naming, glad you guys are on it! 👍🏾
